tests/safety/common.py

```diff
@@ -175,6 +174,53 @@ def test_accel_actuation_limits(self, stock_longitudinal=False):
           self.assertEqual(should_tx, self._tx(self._accel_msg(accel)))
 
 
+class LongitudinalGasBrakeSafetyTest(PandaSafetyTestBase, abc.ABC):
+  MIN_BRAKE: int = 0
+  MAX_BRAKE: Optional[int] = None
+
+  MIN_THROTTLE: int = 1818
```

Comment:
let's use gas for now, it's what the other brands use

Reply:
done